src/main/java/com/laytonsmith/PureUtilities/Point3D.java

```diff
@@ -27,25 +27,10 @@ public Point3D(Point3D other) {
 	}
 
 	/**
-	 * Initializes the X, Y, and Z values to 0.
-	 * Since Point3Ds are immutable, it is recommended to use Point3D.ZERO instead.
-	 *
-	 * @deprecated please use Point3D.ZERO instead
-	 */
-	@Deprecated
-	public Point3D() {
-		this.x = 0;
-		this.y = 0;
-		this.z = 0;
-	}
-
-	/**
 	 * Initializes the X and Y values. Z is initialized to 0.
 	 */
 	public Point3D(double x, double y) {
-		this.x = x;
-		this.y = y;
-		this.z = 0;
+		this(x, y, 0);
```

Comment:
could you please revert this to how it used to be?
http://stackoverflow.com/questions/30001991/performance-should-i-avoid-constructor-delegation
just a tiny tiny optimization.